Automattic/wp-calypso
# Introduce live chat prompt for personal and premium users during cancellation flow

This PR introduces a live chat a/b test to WordPress.com personal and premium users as a second step in the cancellation and remove flows:

![screen shot 2017-04-25 at 11 21 43 am](https://cloud.githubusercontent.com/assets/1926/25365186/6af3821c-29aa-11e7-9645-e6adb46eedb2.png)

Testing

* Create a site with WordPress.com personal or premium plans.
* Select the 'show' variant for the 'chatOfferOnCancel' a/b test.
* Ensure a happychat staging operator is available
* Browse to `/me/purchases`
* Select the plan
* Click 'Remove WordPress.com Premium'
* Choose 'It was too hard to set up my site.' for question one
* Choose any option for question two

The extra step should appear in this case.

This step should only appear if the following conditions are met:

* question one answer was 'too hard'
* WordPress.com Personal or Premium
* Happychat is available
* a/b test variant is 'show'
